airflow-core/src/airflow/ui/src/i18n/locales/he/dag.json

```diff
@@ -0,0 +1,111 @@
+{
+  "allRuns": "כל ההרצות",
+  "blockingDeps": {
+    "dependency": "תלות",
+    "reason": "סיבה",
+    "title": "תלויות החוסמות את תזמון המשימה"
+  },
+  "code": {
+    "bundleUrl": "כתובת החבילה",
+    "noCode": "לא נמצא קוד תכנות",
+    "parsedAt": "נקרא בשעה:"
+  },
+  "extraLinks": "קישורים נוספים",
+  "grid": {
+    "buttons": {
+      "resetToLatest": "אפס לגרסה האחרונה",
+      "toggleGroup": "החלף מצב קבוצה"
+    }
+  },
+  "header": {
+    "buttons": {
+      "dagDocs": "תיעוד DAG"
+    }
+  },
+  "logs": {
+    "noTryNumber": "אין מספר ניסיון",
+    "viewInExternal": "צפה ברישומים ב-{{name}} (ניסיון {{attempt}})"
+  },
+  "overview": {
+    "buttons": {
+      "failedRun_one": "הרצה שנכשלה",
+      "failedRun_other": "הרצות שנכשלו",
+      "failedTask_one": "משימה שנכשלה",
+      "failedTask_other": "משימות שנכשלו",
+      "failedTaskInstance_one": "מופע משימה שנכשל",
+      "failedTaskInstance_other": "מופעי משימות שנכשלו"
+    },
+    "charts": {
+      "assetEvent_one": "אירוע משאב שנוצר",
```

Comment:
Did we decide on "משאב" or "נכס"? (I'm ok with both, but we should be consistent in all translation files)

Reply:
צודק החלטנו על נכס.
משאב השתמשנו עבור Pool